src/components/Dashboard/Flavors.test.js

```diff
@@ -29,4 +45,40 @@ describe('Dashboard <Flavors />', () => {
         .toJSON()
     ).toMatchSnapshot();
   });
+
+  it('can changePage', () => {
+    const component = renderer.create(
+      <Provider store={store}>
+        <RoutedFlavors actions={actions} layoutOptions={defaultLayoutOptions} />
+      </Provider>
+    );
+    const { instance } = component.root.findByType(Flavors);
+
+    expect(instance).toBeDefined();
+    instance.changePage(page);
+  });
```

Comment:
There should be some expectations made after to assert that [setState and actions.requestFlavors](https://github.com/mixnjuice/frontend/pull/43/files/99e5ff2b8d1e9869d58c62f119846312339a7ef5..6672cb715193e110eb18a65755d636f04a554ab4#diff-e9aa3e7b4635a24e56bf93f151e5f8d6R50) are being called.